rotkehlchen/tests/utils/exchanges.py

```diff
@@ -1442,14 +1442,19 @@ def mock_normal_coinbase_query(url, **kwargs):  # pylint: disable=unused-argumen
     raise AssertionError(f'Unexpected url {url} for test')
 
 
-def get_exchange_asset_symbols(exchange: Location) -> set[str]:
+def get_exchange_asset_symbols(
+        exchange: Location,
+        include_null_locations: bool = True,
```

Comment:
again just like many other places I have told you in the past, you should avoid extra if checks like this.

Change the argument to query_suffix with default vaule being `' OR location IS NULL;'` and in the place you want to not include_null_locations call it with `query_suffix=';'`

Simpler and doesn't waste if checks. Adjust the docstring accordingly though